File: guanabara-java/src/com/danillo/algoritmos/lista03/Exercicio33.java
package com.danillo.algoritmos.lista03;

import java.util.Scanner;

public class Exercicio33 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira o valor da casa: ");
        float valorCasa = entrada.nextFloat();
        System.out.println("Insira o seu salário: ");
        float salario = entrada.nextFloat();
        System.out.println("Em quantos anos planeja pagar? ");
        int tempoPagamento = entrada.nextInt();

        int quantidadeParcelas = tempoPagamento * 12;
        float prestacaoMensal = valorCasa/quantidadeParcelas;
        float porcentagemSalario = salario * 30/100;

        if(prestacaoMensal > porcentagemSalario){
            System.out.println("Empréstimo reprovado");
        }else{
            System.out.println("Empréstimo aprovado");
        }
    }
}

File: guanabara-python/src/lista03/exercicio33.py
valor_casa = float(input("Insira o valor da casa: "))
salario = float(input("Insira o salário do comprador: "))
tempo_pagamento = int(input("Em quantos anos o comprador pretende pagar a casa? "))
quantidade_parcelas = tempo_pagamento * 12
valor_parcelas = valor_casa / quantidade_parcelas

if valor_parcelas < (salario * 30)/100:
    print("Empréstimo aprovado.")
else:
    print("Empréstimo reprovado.")
